opencog/openpsi/README.md

```diff
@@ -42,21 +43,8 @@ are used).
 * The demand-update code is undocumented, confusing, and does not
   belong in this directory.
 
-
-### References
-* OpenCog Wiki - [OpenPsi](http://wiki.opencog.org/w/OpenPsi)
```

Comment:
The wiki should be the main/primary reference for OpenPsi documentation -- we should not remove this URL.

Reply:
Ah, I didn't remove them... I moved the references section to the end, as it made more sense to me for it to be there instead of in the middle (also the references section is also referred to later in the document).

Reply:
oh duh